Documentation/Architecture/Package_Manager/Package_Manager.md

```diff
@@ -0,0 +1,31 @@
+# Package Manager
+
+The package manager is intended to highlight what is available from both Microsoft and the community for Mixed Reality Toolkit. It will suplement Unity's built in package management that is available by highlighting [vetted](#package-vetting) resources for developers.
+
+## Package Vetting
+
+Package vetting for showing up in the package manager UI should be done in multiple ways. Naming is subject to change, but the below outlines the general split on how packages will be presented. 
+
+The main goal is that all packages are contained within the Mixed Reality Toolkit Repo, or some separate repo mantained by the same community. This brings clarity to users about where issues can be filed, where to find more information about specific packages, and how they can get their packages added. This will explicitly not display any packages that are not contained within these constraints; while they may be good packages that work well with Mixed Reality Toolkit, they are not mantained within the community and will not be promoted in the same way. Instead, it will be up to the package owners to promote it's use in another way. We may find other promotion techniques if we as a community believe a package provides value but is not part of our ecosystem. 
+
+### Microsoft Packages
```

Comment:
Input and spatial understanding I would promote as coming with a promise of stability and support, which includes from Microsoft and community contributions.

Similarly, Microsoft will contribute some packages which may be more experimental in nature and would not come with long term support. 

Since our grouping of packages together is the contract that MRTK has with an app developer, I think we would name them differently than which organization authored them.

Some ideas:
Production, Supported, Core, Retail, Release
vs
Experimental, Beta, Explorational, Prototype, Community (may still work here)

Reply:
How about "Release" and "Preview"?

Here's why: "Core" can also refer to the implementations of the base level components for MRTK. Input and Spatial Understanding are part of "MRTK Core".

"Community" I feel is overloaded to be both authorship and support level.

"Release" are components/platforms that we will have a strong contract with an app developer stating we'll fix bugs, respond to issues, service requests.

"Preview" are components/platforms that we offer to developers but are "at your own risk" to take a dependency on. This category has flexibility to attract new ideas and find signal. With enough feedback we can promote "Preview" work into the next "Release". But components can also remain "Preview" indefinitely.

Reply:
I think that's a good start. I believe @davidkline-ms  suggest "Standard" and "Extension", but thinking about I like "Release" and "Preview" a bit more.